packages/nodejs-template/package.json

```diff
@@ -15,25 +15,25 @@
     "@bitcoin-computer/lib": "^0.17.2-beta.0"
   },
   "devDependencies": {
-    "@types/bitcoin-computer__lib": "^0.17.1",
-    "@types/mocha": "^9.1.1",
-    "@types/node": "~18.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.10.0",
-    "@typescript-eslint/parser": "^5.10.0",
-    "chai": "^4.3.6",
+    "@types/bitcoin-computer__lib": "^0.17.2",
```

Comment:
Same here, we should remove this devDependency to see if this change effectively work